src/main/java/org/apache/sysds/runtime/controlprogram/federated/FederatedRequest.java

```diff
@@ -74,4 +80,16 @@ public int getNumParams() {
 	public FederatedRequest deepClone() {
 		return new FederatedRequest(_method, new ArrayList<>(_data));
 	}
+
+	public void setCheckPrivacy(boolean checkPrivacy){
+		this.checkPrivacy = checkPrivacy;
+	}
+
+	public void setCheckPrivacy(){
+		setCheckPrivacy(DMLScript.CHECK_PRIVACY);
+	}
+
+	public boolean checkPrivacy(){
+		return checkPrivacy;
+	}
```

Comment:
would `getCheckPrivacy` maybe be more consistent with the `set`? Would be fine with both, just throwing it out there.

Reply:
The name is phrased like a question: "Should I check privacy? Yes or no."

Reply:
If you view it this way I agree that seems more readable :+1: